play/play.go

```diff
@@ -28,13 +28,17 @@ type model struct {
 	question questionModel
 	learn    learn
 
+	parentCallback Callback
+
 	help help.Model
 }
 
 const copyStatus = "Code has copied to clipboard"
 const copyErrStatus = "Code has not copied. Please install xsel, xclip, wl-clipboard or Termux:API"
 
-func New(c challenge.Challenge, width, height int) (tea.Model, tea.Cmd) {
+type Callback func(tea.Msg) (tea.Model, tea.Cmd)
```

Comment:
1. Let's rename it to BackHandler
2. Let's change argument type to tea.WindowSizeMsg

Reply:
Done